fuzzy_logic/fuzzy_operations.py

```diff
@@ -12,7 +12,9 @@
 try:
     import skfuzzy as fuzz
 except ImportError:
-    fuzz = None
+    import sys
+
+    sys.exit()  # This is so the CI doesn't complain about an unknown library
```

Comment:
The commit which made this change https://github.com/TheAlgorithms/Python/blob/master/requirements.txt#L12 is pretty old, let's try installing `scikit-fuzzy` again.

Reply:
Okay, let me know when that's installed so I can remove the try-except

Reply:
You can just uncomment the line in the above link and it will be installed in the next CI run which will happen when you commit those changes.